llama_stack/providers/utils/kvstore/postgres/postgres.py

```diff
@@ -0,0 +1,103 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the terms described in the LICENSE file in
+# the root directory of this source tree.
+
+from datetime import datetime
+from typing import List, Optional
+
+import psycopg2
+from psycopg2.extras import DictCursor
+
+from ..api import KVStore
+from ..config import PostgresKVStoreConfig
+
+
+class PostgresKVStoreImpl(KVStore):
+    def __init__(self, config: PostgresKVStoreConfig):
+        self.config = config
+        self.conn = None
+        self.cursor = None
+
+    async def initialize(self) -> None:
```

Comment:
should probably validate that the table name in the config is sane (doesn't have some hyphens etc which PG won't like)

Reply:
done.